documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+The benefits are following:
+1. Arch Board can review the ApiView when swagger PR is created and no need to wait for SDK owners to create ApiView manually, which saves much time.
+2. Service team can change the swagger in the PR directly if Arch Board has some comments in reviewing ApiView because the swagger PR is not merged.
+
+This Document describes the process of how to onboard DPG with SDK automation pipeline. __This document is targeting for service team.__ 
+
+*If you want to go through details of pipeline implementation, please go to [Integrate DPG and ApiView in SDK Automation Pipeline](Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md).*
+
+## WorkFlow
+
+![workflow-for-service-team](workflow-service-team.png)
+
+__Description:__
+1. Creates a swagger PR with [Guide to design and creation of Data Plane REST API and Client Libraries](https://aka.ms/azsdk/dpcodegen).
+2. If the SDK was generated for this service before , please go to step 4. Otherwise, this is a new service and please go to step 3.
+3. Create branch with name `dpg/<prNumber>` in sdk repository, and add autorest configuration file. Then add a comment `/azp run` in swagger PR.
+   1. `<prNumber>` is the PR number of swagger PR.
+   2. About how to add autorest configuration file, please refer to [Add autorest configuration file in sdk repository](./add-autorest-configuration-file-in-sdk-repository.md).
+4. Swagger PR CI generates SDK and ApiView automatically, and you can find them in swagger PR comments.
+5. Ask Arch Board to review the generated ApiView. (Please refer to the [Guide of Reviewing the Api Design/Definition](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/591/Guide-to-design-and-creation-of-Data-Plane-REST-API-and-Client-Libraries?anchor=ii.-review-the-api-design/definition)).
+6. If Arch Board approves the ApiView, go to step 10. Otherwise, go to step 7.
+7. Update Swagger if needed.
+8. Update autorest configuration file if needed. Here, if there is a branch named `dpg/<prNumber>` in sdk repository, please update it in the branch. Otherwise, create a branch named `dpg/<prNumber>` from `main` branch in sdk repository and update the `autorest.md` or `README.md` in the new created branch.
```

Comment:
Can we put 8 into step 2?

Reply:
Or put this into a seperate note:  if the change of the configuration file is needed, please do......